src/main/flight/pid.c

```diff
@@ -983,12 +986,11 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
         }
 
         // -----calculate I component
-        const float ITermNew = constrainf(ITerm + pidCoefficient[axis].Ki * itermErrorRate * dynCi, -itermLimit, itermLimit);
-        const bool outputSaturated = mixerIsOutputSaturated(axis, errorRate);
-        if (outputSaturated == false || ABS(ITermNew) < ABS(ITerm)) {
-            // Only increase ITerm if output is not saturated
-            pidData[axis].I = ITermNew;
+        // iterm_windup constrains I accumulation, only on pitch and roll, only when < 100
+        if ((axis <= FD_PITCH) && (ITermWindupPointInv < 1.0f)) {
```

Comment:
Why is yaw removed from this check? The yaw actually shows the most accumulation from high setpoints?

Reply:
Because for yaw, I operates like P, and should not be constrained in this way.  It needs to accumulate, even with motors being driven full, to drive the yaw effectively.